pkg/controller/controllerutil/controllerutil_test.go

```diff
@@ -677,27 +677,33 @@ var _ = Describe("Controllerutil", func() {
 			}
 
 			It("should add the finalizer when not present", func() {
-				controllerutil.AddFinalizer(deploy, testFinalizer)
+				Expect(controllerutil.AddFinalizer(deploy, testFinalizer)).To(BeTrue())
```

Comment:
It would be nice to keep old test "as is" without return validation to check for backward compatibility and create an additional tests for the new functionality validation

Reply:
A change was force-pushed that leaves the old tests unchanged and adds the new ones